module-domain/src/main/kotlin/site/yourevents/invitation/service/InvitationService.kt

```diff
@@ -55,4 +56,12 @@ class InvitationService(
 
         return invitation.qrUrl
     }
+
+    override fun deleteInvitation(invitationId: UUID, deleted: Boolean) {
```

Comment:
`deleted` 매개변수가 사용되지 않기에, 해당 부분 제거해도 좋아보입니다!
```suggestion
    override fun deleteInvitation(invitationId: UUID) {
```

Reply:
넵 제거하였습니다!!